app/services/task_service/push_external.rb

```diff
@@ -32,6 +39,8 @@ def request_parameters(request, body)
     def connection
       Faraday.new(url: @account_link.push_url) do |faraday|
         faraday.adapter Faraday.default_adapter
+        faraday.options[:open_timeout] = 5
+        faraday.options[:timeout] = 5
       end
```

Comment:
Given the similar lines in `check_external.rb`, I would prefer the same order and alignment of these three options:

https://github.com/openHPI/codeharbor/blob/dc64015490a1ebaa364688e484a0ef8ae7cb61c7/app/services/task_service/check_external.rb#L43-L48

Reply:
You are right, that makes sense.
Thanks for the (not)-review and sanity checks!

Reply:
You're very welcome! Thanks for your great ongoing work, I am delighted to see these enhancements landing soon 👍